Week3/hanoi/src/App.java

```diff
@@ -3,21 +3,27 @@
 public class App {
   public static void main(String[] args) {
 
-    /* Get num_disks from System.in */
-    int num_disks;
-
+    // Get num of disks from System.in
+    // The number of disks should be more than 0.
+    int numDisks = 0;
```

Comment:
comment for variable

Reply:
also specify it as private variable

Reply:
https://coderanch.com/t/652201/java/private-variable-main-method
I think it is not available.